src/graph/gk_ops.cc

```diff
@@ -0,0 +1,106 @@
+/*!
+ *  Copyright (c) 2020 by Contributors
+ * \file graph/gk_ops.cc
+ * \brief Graph operation implemented in GKlib
+ */
+
+#if !defined(_WIN32)
+#include <GKlib.h>
+#endif  // !defined(_WIN32)
+
+#include <dgl/graph_op.h>
+
+namespace dgl {
+
+#if !defined(_WIN32)
+
+namespace {
+
+gk_csr_t *Convert2GKCsr(CSRPtr csr, bool is_row) {
+  // TODO(zhengda) The conversion will be zero-copy in the future.
+  const aten::CSRMatrix mat = csr->ToCSRMatrix();
+  const dgl_id_t *indptr = static_cast<dgl_id_t*>(mat.indptr->data);
+  const dgl_id_t *indices = static_cast<dgl_id_t*>(mat.indices->data);
+
+  gk_csr_t *gk_csr = gk_csr_Create();
+  gk_csr->nrows = mat.num_rows;
+  gk_csr->ncols = mat.num_cols;
+  size_t nnz = csr->NumEdges();
```

Comment:
I suggest to use int64_t here.

Reply:
actually, the return type of NumEdges is `uint64_t`.

Reply:
shall we use uint64_t nnz directly?

Reply:
i did it.